CHANGELOG.md

```diff
@@ -4,11 +4,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-## [0.47.0] - 2022-03-29
```

Comment:
We don't align patch version of the chart with patch version of the image. So chart version should still be 0.47.0